ozpricechecker/pricefinderapp/tests/models/test_product.py

```diff
@@ -18,6 +20,20 @@ def test_str_no_name(self):
     def test_str_name(self):
         product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
         self.assertEqual(str(product), 'Woolworths - Super Tomatoes')
+    
+    def test_latest_prod_price_date(self):
+        product = Product.objects.create(store=self.store, prod_url='/1234/Tomatoes', name='Super Tomatoes')
+        productPrice1 = ProductPrice.objects.create(product=product, price=10)
+        time.sleep(1)
```

Comment:
Don't use sleep, set the time directly instead

Reply:
Agreed once we allow setting the Data manually on Product Price we need to remove the sleep statements.